spec/Body/Body.vspec

```diff
@@ -242,7 +242,7 @@ Lights.DirectionIndicator:
 ##
 Mirrors:
   type: branch
-  instances: ["Left", "Right"]
+  instances: ["DriverSide", "PassengerSide"]
   description: All mirrors.
```

Comment:
To cover odd vehicles like McLaren F1 and motorbikes should we possibly add a comment:
`For vehicles where driver position is Middle the left mirror is considered to be the DriverSide the mirror`